config/mfDefs.gem

```diff
@@ -43,8 +43,8 @@ UserDynamicLinkFlags =
 IncludeDirs+=$(XDAQ_ROOT)/include
 
 ## Python
-PYTHON_VERSION ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
-PYTHON_INCLUDE_PREFIX ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
+PYTHON_VERSION ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
+PYTHON_INCLUDE_PREFIX ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
```

Comment:
I suggested to postpone this, because I'm not 100% sure it will work with suggested change at cosmicstandtif